pkg/controller/metrics/metrics.go

```diff
@@ -97,8 +113,35 @@ func (mc *Calculator) Start(stopCh <-chan struct{}) error {
 				installedTotal = installedTotal + 1
 			}
 		}
+		// jobs metrics
+		jobs := &batchv1.JobList{}
+		err = mc.Client.List(context.Background(), &client.ListOptions{}, jobs)
+		if err != nil {
+			log.WithError(err).Error("error listing install jobs")
+		}
+		mcLog.Infof("calculating jobs metrics")
+		installJobsTotal := 0
+		uninstallJobsTotal := 0
+		// legacyJobs is the variable that tells us about the number of old jobs that might exist in production
+		legacyJobs := 0
+		// calculating number of install jobs, uninstall jobs and legacy jobs using labels
+		for _, job := range jobs.Items {
+			if _, ok := job.Labels[installJobLabel]; ok {
+				installJobsTotal = installJobsTotal + 1
```

Comment:
The cards actually specified "running" which will help make this a bit more useful, as we know the number of install jobs will pretty much always be equal to the number of cluster deployments. Tracking the running install/uninstall jobs helps us understand the load a little better.

Lets include "running" in the metric names as well, hive_install_jobs_running_total perhaps.